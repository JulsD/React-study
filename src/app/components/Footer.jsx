import React from 'react';
import Logo from './Logo';

class Footer extends React.Component {

  render() {
    const style = {
      width: '100%',
      backgroundColor: '#474747'
    };
    return (
      <footer style={style}>
        <Logo />
      </footer>
    );
  }

}

export default Footer;
