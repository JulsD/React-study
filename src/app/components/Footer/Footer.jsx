import React from 'react';

import { Logo } from './../';
import common from './../../../styles/common.css';
import styles from './footer.css';

class Footer extends React.Component {

  render() {
    return (
      <footer className = {styles.root}>
        <div className = {common.wrapper}>
          <Logo />
        </div>
      </footer>
    );
  }

}

export default Footer;
