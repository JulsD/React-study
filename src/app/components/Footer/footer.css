.root {
  width: 100%;
  background-color: #474747;
}