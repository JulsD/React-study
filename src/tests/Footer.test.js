import React from 'react';
import Footer from '../app/components/Footer';

test('Footer should render a <footer>', () => {
  const wrapper = shallow(
    <Footer />
  );
  expect(wrapper).toMatchSnapshot();
});
