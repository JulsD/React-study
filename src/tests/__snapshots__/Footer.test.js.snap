// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render a <footer> 1`] = `
<footer
  className="root"
>
  <Wrapper>
    <Logo />
  </Wrapper>
</footer>
`;
